ray-operator/controllers/ray/common/pod.go

```diff
@@ -34,7 +34,7 @@ const (
 	EnableInitContainerInjectionEnvKey = "ENABLE_INIT_CONTAINER_INJECTION"
 )
 
-var log = logf.Log.WithName("RayCluster-Controller")
+var log = logf.Log.WithName("controllers").WithName("RayCluster")
```

Comment:
Make it consistent with 
https://github.com/ray-project/kuberay/blob/c55f3cc6778aa8d32f4997abdf50fda941a39886/ray-operator/controllers/ray/raycluster_controller.go#L105